operator/example/kustomization.yaml

```diff
@@ -1,9 +1,6 @@
 resources:
   - testroute.yaml
 
-generatorOptions:
-  disableNameSuffixHash: true
-
 configMapGenerator:
   - name: testroute-xml
```

Comment:
:question: Should the XML ConfigMap also have its hash suffix disable?

Reply:
Yeah good catch